### PR TITLE
Remove apparent grok dependencies

### LINKED ISSUE #11: Remove dependency on five.grok

### PR DESCRIPTION
**Pull request by rodfersou**
closes #11

### CODE REVIEW

**Comment by hvelarde** [comment]
IIRC, you have to remove `plone.directives.form` and `plone.directives.dexterity` also:

```bash
$ grep grok bin/instance 
    '/home/hvelarde/.buildout/eggs/grokcore.view-2.8-py2.7.egg',
    '/home/hvelarde/.buildout/eggs/five.grok-1.3.2-py2.7.egg',
    '/home/hvelarde/.buildout/eggs/grokcore.security-1.6.2-py2.7.egg',
    '/home/hvelarde/.buildout/eggs/grokcore.component-2.5-py2.7.egg',
    '/home/hvelarde/.buildout/eggs/grokcore.viewlet-1.11-py2.7.egg',
    '/home/hvelarde/.buildout/eggs/grokcore.site-1.6.1-py2.7.egg',
    '/home/hvelarde/.buildout/eggs/grokcore.annotation-1.3-py2.7.egg',
```

File: src/sc/periodicals/browser/periodical.py [hunk 1]
```diff
@@ -1,56 +1,46 @@
...
-    grok.layer(IPeriodicalLayer)
-    grok.require('zope2.View')
-    grok.viewletmanager(IAboveContent)
 
     def update(self):
-        self.context = aq_inner(self.context)
+        super(PeriodicalHeader, self).update()
+        self.setup()
+
+    def setup(self):
```
Comment (hvelarde) [inline src/sc/periodicals/browser/periodical.py]: please remove this method; let's follow the conventions: just `update()` and `render()` and just use `super()` if you're going to reuse some of the code there.

https://docs.plone.org/develop/plone/views/viewlets.html#viewlet-behavior

File: src/sc/periodicals/browser/configure.zcml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      class=".periodical.View"
+      layer="..interfaces.IPeriodicalLayer"
+      />
+
+  <browser:viewlet
+      name="sc.periodicals.periodicalheader"
+      manager="plone.app.layout.viewlets.interfaces.IAboveContent"
+      class=".periodical.PeriodicalHeader"
+      permission="zope2.View"
+      template="templates/periodicalheader.pt"
```
Comment (hvelarde) [inline src/sc/periodicals/browser/configure.zcml]: please move the templates to the browser folder; let's keep it plain.

File: src/sc/periodicals/browser/periodical.py [hunk 3]
```diff
@@ -1,56 +1,46 @@
...
-class View(dexterity.DisplayForm):
-    """ Default view.
-    """
-    grok.context(IPeriodical)
-    grok.require('zope2.View')
-    grok.layer(IPeriodicalLayer)
-
-    def update(self):
-        self.context = aq_inner(self.context)
+    index = ViewPageTemplateFile('templates/view.pt')
```
Comment (hvelarde) [inline src/sc/periodicals/browser/periodical.py]: please move the templates to the browser folder; let's keep it plain.

File: src/sc/periodicals/configure.zcml [hunk 4]
```diff
@@ -10,6 +9,7 @@
   <i18n:registerTranslations directory="locales" />
   <includeDependencies package="." />
```
Comment (hvelarde) [inline src/sc/periodicals/configure.zcml]: please don't autoload dependencies; list explicitly what is needed.

Comment (rodfersou) [inline src/sc/periodicals/configure.zcml]: [other packages have the same directive](https://github.com/collective/collective.cover/blob/master/src/collective/cover/configure.zcml#L13), and if I remove it the tests fail.

Comment (rodfersou) [inline src/sc/periodicals/configure.zcml]: Found other example.. done

**Pull request merged**